yarn-project/noir-contracts/src/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -187,7 +187,7 @@ contract InclusionProofs {
         // 5.c) Prove that the low nullifier is pointing "over" the nullifier to prove that the nullifier is not
         // included in the nullifier tree
         assert(
-            full_field_greater_than(witness.leaf_data.next_value, nullifier),
+            full_field_greater_than(witness.leaf_data.next_value, nullifier) | (witness.leaf_data.next_value == 0),
```

Comment:
For good measure, you should probably check both `next_value` and `next_idx` or just the index to follow what we are doing in the data here.

https://docs.aztec.network/concepts/advanced/data_structures/indexed_merkle_tree#non-membership-proof